src/strategies_and_params/poststepstrategy.jl

```diff
@@ -24,8 +24,9 @@ abstract type PostStepStrategy end
 """
     post_step(::PostStepStrategy, ::ReplicaState) -> kvpairs
 
-Compute statistics after FCIQMC step. Should return a tuple of `:key => value` pairs. See
-also [`PostStepStrategy`](@ref).
+Compute statistics after FCIQMC step. Should return a tuple of `:key => value` pairs. 
+This function is only called every `reporting_interval` steps, as defined by the `ReportingStrategy`. 
```

Comment:
```suggestion
This function is only called every [`reporting_interval`](@ref) steps, as defined by the `ReportingStrategy`. 
```
Since this is a function, we can link to its docs.